src/cannon-es-debugger.ts

```diff
@@ -240,7 +239,7 @@ export default function cannonDebugger(
 
     let meshIndex = 0
 
-    for (const body of bodies) {
+    for (const body of world.bodies) {
```

Comment:
Considering that you only use the world to access the bodies, it seems a bit overkill to bring the entire world into the debugger.

In `use-cannon` the world lives in a work and we don't have access to it so it's quite difficult to have this as a requirement for using the debugger.

Reply:
I preferred matching the signature to the original CannonDebugRenderer.

We can just mock the world (like we're doing now) in use-cannon.